swupd/swupd.go

```diff
@@ -66,6 +94,9 @@ func (s *SoftwareUpdater) Verify(version string, mirror string) error {
 		"swupd",
 		"verify",
 	}
+
+	args = s.setExtraFlags(args)
+
 	if mirror != "" {
 		args = append(args, fmt.Sprintf("--url=%s", mirror))
```

Comment:
Is there any conflict in settings --versionurl --contenturl and --url?

Reply:
nope